### PR TITLE
@truffle/hdwallet-provider: additional ignores in .npmignore

### PR DESCRIPTION
**Pull request by MrRefactoring**
Before:
before
After:
after

### CODE REVIEW

File: packages/hdwallet-provider/.npmignore [hunk 1]
```diff
@@ -1,2 +1,7 @@
 .idea
 .vscode
+test
+.travis.yml
```
Comment (CruzMolina) [inline packages/hdwallet-provider/.npmignore]: Oh, there'a  `.travis.yml` in here?!!

Are you open to removing it as part of this PR?

Comment (MrRefactoring) [reply packages/hdwallet-provider/.npmignore]: Why do we need .travis.yml in the final build of the package?

Comment (CruzMolina) [reply packages/hdwallet-provider/.npmignore]: Ah, I miscommunicated. I was referring to the pkg itself.

I didn't know there was a `.travis.yml` file in the pkg. We no longer use it (probably leftover from before this pkg was a part of the monorepo). Are you open to removing the `travis.yml` from this pkg as part of the PR?

Comment (MrRefactoring) [reply packages/hdwallet-provider/.npmignore]: Yeah sure

Comment (MrRefactoring) [reply packages/hdwallet-provider/.npmignore]: Done

**Comment by gnidan** [comment]
So lean

**Pull request merged**